services/director-v2/tests/integration/02/test_dynamic_sidecar_nodeports_integration.py

```diff
@@ -621,17 +621,18 @@ async def _fetch_data_via_data_manager(
     save_to.mkdir(parents=True, exist_ok=True)
 
     assert (
-        await data_manager.exists(
+        await data_manager.state_metadata_entry_exists(
```

Comment:
actually this name is pretty weird.
I think I do not want to know as the client, whether it is saved as an archive or a folder or even an encoded whatever...

Reply:
was moved and hidden away